recon_surf/paint_cc_into_pred.py

```diff
@@ -79,22 +80,22 @@ def argument_parse():
 
 
 def paint_in_cc(pred: npt.ArrayLike, aseg_cc: npt.ArrayLike) -> npt.ArrayLike:
-    """Paint corpus callosum segmentation into prediction.
+    """
+    Paint corpus callosum segmentation into prediction.
 
     Note, that this function modifies the original array and does not create a copy.
 
     Parameters
     ----------
     pred : npt.ArrayLike
-        Deep-learning prediction
+        Deep-learning prediction.
     aseg_cc : npt.ArrayLike
-        Aseg segmentation with CC
+        Aseg segmentation with CC.
 
     Returns
     -------
     pred
-        Prediction with added CC
-
+        Prediction with added CC.
```

Comment:
we should not use "pred"/"prediction" here change to asegdkt in this file and create a TODO for another student assistant to rename the file (paint_cc_into_asegdkt or similar) to rename the file and functions.

Reply:
Replaced instances of 'pred' with 'asegdkt'.
Added a TODO comment in last line.